D/DBCtoDBF/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+#
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DBCtoDBF"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lego-yaw/DBCtoDBF.git", "ad4a26e2c3da9d1499b7f71d5e4859dc0ad708bd")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd DBCtoDBF/
+mkdir -p ${prefix}/share/licenses/DBCtoDBF
+cp LICENSE ${prefix}/share/licenses/
+cp LICENSE ${prefix}/share/licenses/DBCtoDBF/
+cd SRC/
+make
+mkdir -p ${prefix}/bin/
+cp dbc2dbf  ${prefix}/bin/
+make test
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "freebsd"; )
+]
+
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("dbc2dbf", :dbc2dbf)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"13.2.0")
```

Comment:
https://github.com/JuliaPackaging/Yggdrasil/blob/afe8e6d6949885f0a01f1ddfaa35033739269529/CONTRIBUTING.md#compatibility-tips

Reply:
ok

Reply:
You didn't change the version of gcc

Reply:
You still haven't changed this.